.twistlockignore-amd64-static

```diff
@@ -3,4 +3,5 @@ CVE-2023-24538 # we are not affected by html templating
 CVE-2023-24540 # we do not do any whitespace cleaning
 CVE-2023-29402 # we are not in control of the building pipeline for external components
 CVE-2023-29404 # we are not in control of the building pipeline for external components
-CVE-2023-29405 # we are not in control of the building pipeline for external components
\ No newline at end of file
+CVE-2023-29405 # we are not in control of the building pipeline for external components
+CVE-2023-33246 # we are not in control of the building pipeline for external components
```

Comment:
I would change the comment as this is actually only relevant for servers and we are a client AFAIK from an MQ point of view.